### PR TITLE
Install podman where required

### PR DESCRIPTION
**Pull request by nocturnalastro**

### CODE REVIEW

File: roles/setup_http_store/tasks/main.yml [hunk 1]
```diff
@@ -46,14 +50,14 @@
         volumes:
           - "{{ http_data_dir }}:/var/www/html:z"
       register: container_info
-  become: true
 
 - name: Setting facts about container
```
Comment (crwr45) [inline roles/setup_http_store/tasks/main.yml:54]: This seems to be the only task in the file not in a `become: true` block, and there's no reason for it not to be IMO.

**Review by crwr45 — APPROVED** [review]

File: roles/setup_http_store/tasks/main.yml [hunk 2]
```diff
@@ -1,39 +1,43 @@
...
+
     - name: Create httpd container
       containers.podman.podman_pod:
         name: "{{ http_store_pod_name }}"
         publish:
           - 80:8080
       register: pod_info
 
-    - debug: # noqa unnamed-task
+    - debug:  # noqa unnamed-task
```
Comment (arjuhe) [inline roles/setup_http_store/tasks/main.yml:40]: Extra space here.

**Review by arjuhe — COMMENTED** [review]
I am still not a fan of using a block as a way to wrap become.

**Review by crwr45 — APPROVED** [review]

**Review by arjuhe — APPROVED** [review]
+1

**Pull request merged**